config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1264,55 +1264,48 @@ presubmits:
             cpu: 4
             memory: 6Gi
 
-  # This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently alpha).
-  # crio >= 1.23.0 has the necessary support for Container Device Interface (CDI).
-  - name: pull-kubernetes-node-crio-dra
-    cluster: k8s-infra-prow-build
+  # This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently alpha)
+  # on a kind cluster with containerd updated to a version with CDI support.
```

Comment:
should we create a TODO comment with the corresponding task to clean this up in future when containerd latest will support it?

Reply:
I think we'll keep working on this job either way, also without an explicit TODO.